### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by pptino** [microsoft/vscode-azureappservice#2440]
Does this occur consistently? 
Repro steps:

1.
2.

Action: appService.Deploy
Error type: TypeError
Error Message: Cannot read properties of undefined (reading 'createClient')

Version: 0.24.8
OS: darwin
OS Release: 22.3.0
Product: Visual Studio Code
Product Version: 1.76.2
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:2:3774915
fulfilled extension.bundle.js:2:3773586
```

</details>

**Comment by alexweininger** [comment]
The fix for this bug has been merged and is waiting to be released in the Azure Resources extension. https://github.com/microsoft/vscode-azureresourcegroups/pull/642

**Comment by alexweininger** [comment]
This issue has been fixed by the [0.7.0 release of the Azure Resources extension](https://github.com/microsoft/vscode-azureresourcegroups/releases/tag/v0.7.0). Please update the Azure Resources extension to version 0.7.0.
